dashboards/server.jsonnet

```diff
@@ -0,0 +1,180 @@
+local template = grafana.template;
+local dashboard = grafana.dashboard;
+local transformation = grafana.transformation;
+
+local grafana7 = import 'grafonnet-7.0/grafana.libsonnet';
+local stat = grafana7.panel.stat;
+local table = grafana7.panel.table;
+local graph = grafana7.panel.graph;
+
+local astimeseries(x, format='short', stack=false) = x {
+  type: 'timeseries',
+  options: {
+    tooltip: {
+      mode: 'multi',
+    },
+  },
+  fieldConfig: {
+    defaults: {
+      unit: format,
+      custom: {
+        stacking: {
+          mode: if stack then 'normal' else 'none',
+        },
+      },
+    },
+  },
+};
+
+dashboard.new(
+  'Infra Server',
+  schemaVersion=16,
+  timezone='utc',
+  uid='512571c8648d41f68e7f595261d51a71',
+)
+.addTemplates([
+  template.datasource(
+    'datasource',
+    'prometheus',
+    'Prometheus',
+  ),
+  template.new(
+    'job',
+    '$datasource',
+    'label_values(build_info{container="server"}, job)',
+    label='Job',
+    refresh='time',
+  ),
+  template.new(
+    'instance',
+    '$datasource',
+    'label_values(build_info{container="server", job="$job"}, instance)',
+    label='Instance',
+    refresh='time',
+    includeAll=true,
+    multi=true,
+  ),
+  template.new(
+    'http_request_path',
+    '$datasource',
+    'label_values(http_request_duration_seconds_count{job="$job", instance=~"$instance"}, path)',
+    label='HTTP Request',
+    refresh='time',
+  ),
+  template.new(
+    'http_request_method',
+    '$datasource',
+    'label_values(http_request_duration_seconds_count{job="$job", instance=~"$instance", path="$http_request_path"}, method)',
+    label='HTTP Method',
+    refresh='time',
+    includeAll=true,
+    multi=true,
+  ),
+])
+.addPanels([
+  stat.new(datasource='$datasource')
+    .setGridPos(x=0, y=0, h=6, w=24)
+    .addTarget(prometheus.target(
+      'sum(build_info{job="$job"})',
+      legendFormat='Num. of Replicas',
+    ))
+    .addTarget(prometheus.target(
+      'sum by (db_name) (go_sql_open_connections{job="$job"})',
+      legendFormat='Num. of Open {{ db_name }} Connections',
+    ))
+    .addTarget(prometheus.target(
+      'sum(infra_providers{job="$job"}) / sum(build_info{job="$job"})',
+      legendFormat='Num. of Providers',
+    ))
+    .addTarget(prometheus.target(
+      'sum(infra_users{job="$job"}) / sum(build_info{job="$job"})',
+      legendFormat='Num. of Users',
+    ))
+    .addTarget(prometheus.target(
+      'sum(infra_groups{job="$job"}) / sum(build_info{job="$job"})',
+      legendFormat='Num. of Groups',
+    ))
+    .addTarget(prometheus.target(
+      'sum(infra_destinations{job="$job"}) / sum(build_info{job="$job"})',
+      legendFormat='Num. of Destinations',
+    ))
+    .addTarget(prometheus.target(
+      'sum(infra_grants{job="$job"}) / sum(build_info{job="$job"})',
+      legendFormat='Num. of Grants',
+    )),
+  astimeseries(graph.new(title='HTTP Request Latency', datasource='$datasource')
```

Comment:
Not a blocker at all, but updating this graph seems like it will be difficult, it seems pretty complicated